lib/src/main/java/uk/gov/di/ipv/core/library/service/CiStorageService.java

```diff
@@ -100,25 +132,186 @@ public void submitMitigatingVcList(
     public List<ContraIndicatorItem> getCIs(
```

Comment:
I think, can name getCisFromStorage

Reply:
CIStorage is now renamed as CIMIT.
this call return CI's